### PR TITLE
quotes workflow name since the name includes a comma

### PR DESCRIPTION
**Pull request by gilzow**
## Why

PR #4244 introduced an issue with PR comments workflow

## What's changed

TL;DR - adds quotes to the name of dependency workflow

The workflow that adds PR comments (comment-on-pr.yaml) is triggered based on the conclusion of the (formerly named) get-pr-info,yaml workflow. Github uses the `name` property of the parent workflow to identify it. I changed that property to `Get PR URL, start dependent jobs` to more accurately reflect what it does, but now the comment workflow sees that as two different workflows. Quoting the workflow `name` corrects the issue.

### CODE REVIEW

**Review by chadwcarlson — APPROVED** [review]

**Comment by gilzow** [comment]
once you merge #4232 you can update this PR and then it should automerge (the broken redirections that #4232 updates are blocking this PR)

**Pull request merged**
